src/main/resources/openapi.yaml

```
@@ -0,0 +1,67 @@
+openapi: 3.0.3
+info:
+  title: Coin Trade API
+  version: 1.0.0
+  description: 코인 거래소 API
+
+paths:
+  /coins/search:
```

```suggestion
  /v1/coins:
```
시작부터 버저닝을 고려하는게 좋습니다.